src/ios/HealthKit.m

```diff
@@ -860,95 +874,104 @@ - (void) querySampleType:(CDVInvokedUrlCommand*)command {
   }];
 }
 
-- (void) queryCorrelationType:(CDVInvokedUrlCommand*)command {
```

Comment:
Wow, looks exactly like that! :open_mouth: 

I should have restored it now. Thank you!